Backend/src/test/java/com/praisesystem/backend/SignatureDecodingTest.java

```diff
@@ -20,35 +25,73 @@
             "\n" +
             "NONCE:\n" +
             "NONCE123";
+        
+    private static final String MESSAGE_2 = "SIGN THIS MESSAGE TO LOGIN TO PRAISE.\n" +
+            "\n" +
+            "ADDRESS:\n" +
+            "0x0bfBC654e57f9db86D422D5666627dd32b551D55\n" +
+            "\n" +
+            "NONCE:\n" +
+            "NONCE987";
 
     private static String SIGNATURE = "0x80a922517b1b18b9c73874d3b08964beaf23e53ef14ccd5cff8913c9dd9f6cf345a7ace75ae63b303a9cdac2bd0b9421d3c76337fbb26c8b55cba3688fac0bf01c";
     private static final String ADDRESS = "0xf592eE0E3a20Ddd65882E0fE6bFBB4B465A98Ae4";
 
     @Test
-    void test3() throws SignatureException {
-        byte[] signature = Numeric.hexStringToByteArray(SIGNATURE);
+    void getPubKeyFromMessageAndSignatureTest() throws SignatureException {
+
+        String result_1 = AuthParamsValidator.getPubKeyFromMessageAndSignature(MESSAGE, SIGNATURE);
+        Assertions.assertEquals(ADDRESS, result_1);
```

Comment:
assertj         Assertions.assertThat("abc").isEqualTo("abc");

just put Assertions.assertThat(something).is... < and you will find tons of very cool methods :)